modules/learn/pages/clusters-and-availability/xdcr-conflict-resolution.adoc

```diff
@@ -57,6 +76,11 @@ If both document-versions have the same timestamp-value, the conflict is resolve
 . Expiration (TTL) value
 . Document flags
 
+[#eccv-true-in-timestamp-based-conflict-resolution]
+=== When enableCrossClusterVersioning is true for a timestamp-based conflict resolution process
+
+When the `enableCrossClusterVersioning` property is set to `true`, the hybrid logical vector (HLV) metadata located in the document xattr (system created extended attributes) of the source and the target, is also used in the timestamp-based conflict resolution processing. The features or options used in the XDCR process determine the specific ways of using the HLV metadata. For more information on the `enableCrossClusterVersioning` property and the HLV metadata, see xref:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc[XDCR enableCrossClusterVersioning].
```

Comment:
Remove lines 79-82.

Add a line that says:
When Cross Cluster Versioning is enabled, the hybrid logical vector (HLV) metadata in the source and target document's xattr, is also used in the conflict resolution processing.